io.httpie.HTTPie.yml

```diff
@@ -0,0 +1,75 @@
+app-id: io.httpie.HTTPie
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+separate-locales: false
+tags:
+  - proprietary
+command: httpie
+rename-icon: httpie
+rename-desktop-file: httpie.desktop
+finish-args:
+  - --socket=x11
+  - --socket=wayland
```

Comment:
does it really support wayland? Usually electon apps do not.

Reply:
Yeah! If you pass `--enable-features=UseOzonePlatform --ozone-platform=wayland ` as flag to Electron Application(just tested with HTTPie) , it runs as Wayland Application instead of X11.

Reply:
the NO.

It doesn't.

```suggestion
```

Reply:
@hfiguiere , I corrected it.